src/util/Logger.js

```diff
@@ -13,7 +13,8 @@
 define(function () {
 
   function log(level, args) {
-    if (window.console) {
+    // Only log statements in development mode
+    if (DEBUG) {
```

Comment:
Based on which versions of the Widget [we support](https://help.okta.com/en/prod/Content/Topics/Miscellaneous/Platforms_Browser_OS_Support.htm), we don't need the `window.console` check anymore.